doc/services/portability/posix/conformance/index.rst

```diff
@@ -109,7 +109,7 @@ POSIX System Interfaces
     _XOPEN_CRYPT, -1,
     _XOPEN_REALTIME, -1,
     _XOPEN_REALTIME_THREADS, -1,
-    :ref:`_XOPEN_STREAMS<posix_option_xopen_streams>`, -1, :kconfig:option:`CONFIG_NET_SOCKETS`
+    :ref:`_XOPEN_STREAMS<posix_option_xopen_streams>`, -1, :kconfig:option:`CONFIG_NET_SOCKETS` (will fail with ``ENOSYS``:ref:`†<posix_undefined_behaviour>`)
```

Comment:
Once all features for `_XOPEN_STREAMS` are available (at the interface level), this can be changed to `200809L` instead of `-1`. But the reference to `CONFIG_NET_SOCKETS` is invalid, so please remove it.

```suggestion
    :ref:`_XOPEN_STREAMS<posix_option_xopen_streams>`, -1, :ref:`†<posix_undefined_behaviour>`
```